common/views/components/PageLayout/PageLayout.tsx

```diff
@@ -162,14 +162,22 @@ const PageLayoutComponent: FunctionComponent<Props> = ({
     <>
       <Head>
         <title>{fullTitle}</title>
-        <meta name="description" content={description} />
+        <meta key="metadescription" name="description" content={description} />
```

Comment:
Would it be useful to add a comment linking to the next issue describing why we're adding `key` here?

Reply:
I think not here, as the Next issue is saying that this _should_ work but doesn't. It's [a recommendation from their official documentation](https://nextjs.org/docs/pages/api-reference/components/head#avoid-duplicated-tags) for `next/head`, so "normal". 

If we wanted to keep track of that issue I could add it to where we load the GTM script and explain that's why we're not removing it?